config/locales/en.yml

```diff
@@ -751,7 +751,7 @@ en:
             fixed_fees: "Fixed fees"
             fee_type: "Fee type"
             total: "Total"
-            prompt_text_html: Check the <a href="https://www.gov.uk/government/publications/graduated-fee-calculators" target="_blank" rel="external" title="Graduated Fee Calculator">graduated fee calculators</a> for help
+            prompt_text_html: Check the <a href="https://www.gov.uk/government/publications/graduated-fee-calculators" target="_blank" rel="external" title="Graduated Fee Calculator">graduated fee calculators</a> for help.
```

Comment:
**[not a blocker]** Seems like this wording is the same across all different types of claim (we're repeating the same wording for each claim type that requires it).

💭  Consider this instead:

```ruby
prompt_text_html: &graduated_fee_prompt_content Check the <a
href="https://www.gov.uk/government/publications/graduated-fee-calculators" target="_blank" rel="external" title="Graduated Fee Calculator">graduated fee calculators</a> for help.
...
graduated_prompt_text: *graduated_fee_prompt_content
...
interim_fees_prompt_text_html: *graduated_fee_prompt_content
...
transfer_fees_prompt_text_html: *graduated_fee_prompt_content
```

This should work, and avoid repeating the same content across multiple prompts.

Reply:
cool .. will do like above.